torch/onnx/_internal/fx/type_utils.py

```diff
@@ -44,6 +44,15 @@ def from_sym_value_to_torch_dtype(sym_value: SYM_VALUE_TYPE) -> torch.dtype:
     return _SYM_TYPE_TO_TORCH_DTYPE[type(sym_value)]
 
 
+def is_optional_onnx_dtype_str(onnx_type_str: str) -> bool:
+    optional_onnx_dtype_str = {
```

Comment:
nit: I wonder if we should cache `optional_onnx_dtype_str ` or define it instead of recomputing every time.

Reply:
We should. I missed this.